modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-ibm-storage_{context}"]
+= Setting up {ibm-cloud-title} Object Storage
+
+You can set up {ibm-cloud-title} Object Storage on the command line.
+
+.Prerequisites
+
+* You have installed the latest version of the OpenShift CLI. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
```

Comment:
```suggestion
* You have installed the latest version of `oc`. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
```

Reply:
If you use the attribute then you can just refer to the CLI as `oc`.

Reply:
Good point. Thank you for the suggestion. Applied.